src/components/Card.js

```diff
@@ -8,25 +8,33 @@ export default class Card extends Component {
     "description": PropTypes.string.isRequired,
     "reverseDescription": PropTypes.string.isRequired,
     "faceUp": PropTypes.bool.isRequired,
-    "x": PropTypes.number.isRequired,
-    "y": PropTypes.number.isRequired
+    "isReversed": PropTypes.bool.isRequired,
 
   }
 
+  cardDescription() {
+    let desc = {}
+    desc = (this.props.isReversed === true) ?  this.props.description : this.props.reverseDescription
```

Comment:
cant you just return this last statement? why do you need to return it on a second line?